_posts/2020-09-16-creating-a-snap-chat-mini.markdown

```diff
@@ -0,0 +1,95 @@
+---
+layout: post
+title: "What we Learned Developing Snapchat Minis (Part 1)"
+date: 2020-09-16 12:00:00
+categories: ["learning"]
+author: arieljuod
+---
+
+Over the last few months, we developed a couple of (Snapchat Minis)[https://www.ombulabs.com/blog/software-development/snap-minis.html]. Minis are small static web apps that are run inside a webview within the Snapchat native app.
+
+One important part of the development was to make sure our mini works well across different devices, especially making sure things work the same in both Android's webview (Chrome by default) and iOS's webview (Safari).
+
+In this post I'll talk about a few things to keep in mind when working with the Span canvas SDK and in the next one I'll talk about some common and known issues you'll encounter when making your app cross-mobile-browser compatible.
+
+<!--more-->
+
+## Working with the SDK
+
+Snapchat provides a JavaScript SDK that you'll have to include in your project. This SDK works as a bridge between the native app and your web app and lets you trigger native functionalities (like sharing on chat and camera), it gives you information about the current user and session, and also sends different events so you can react to them (you can detect if a user takes a screenshot or changes the volume for example).
```

Comment:
```suggestion
Snapchat provides a JavaScript SDK that you'll have to include in your project. This SDK works as a bridge between the native app and your web app and lets you trigger native functionalities (like sharing on chat and camera), gives you information about the current user and session, and also sends different events so you can react to them (you can detect if a user takes a screenshot or changes the volume for example).
```

Reply:
Should I also change the `Snapchat` to `Snap` here? (and everywhere I say Snapchat?)

Reply:
Oh I got it, it's just for `Snap Mini`